filebeat/input/redis/harvester.go

```diff
@@ -155,7 +155,9 @@ func (h *Harvester) Run() error {
 				"redis": common.MapStr{
 					"slowlog": subEvent,
 				},
-				"read_timestamp": common.Time(time.Now().UTC()),
+				"event": common.MapStr{
+					"created": common.Time(time.Now().UTC()),
```

Comment:
I think we don't need `common.Time` anymore.

Reply:
You mean just using `time.Now().UTC()` instead? Did something change recently that we don't need it anymore? We use `common.Time` still quite a lot in the code base.

Reply:
Thanks for the details. +1 on removing all the current usage.

I removed common.Time and UTC conversion in the most recent commit.